scripts/openshiftci-presubmit-unittests.sh

```diff
@@ -5,8 +5,7 @@ set -e
 # show commands
 set -x
 
-export ARTIFACTS_DIR="/tmp/artifacts"
-export CUSTOM_HOMEDIR=$ARTIFACTS_DIR
+export CUSTOM_HOMEDIR=$ARTIFACT_DIR
```

Comment:
We also need to change the env below at https://github.com/openshift/odo/blob/9d246db05bc5cf434e3cc74e88e10e0e47b66c03/scripts/openshiftci-presubmit-unittests.sh#L21

Reply:
I think we don't need an extra env var CUSTOM_HOMEDIR for the assignment, instead we can directly use ARTIFACTS_DIR where ever we need.